src/js/view/progressbar.js

```diff
@@ -28,6 +28,14 @@ export default function ProgressBar(elements) {
                 durationElement.innerText = '--:--';
                 sliderElem.style.width = '0%';
                 mediaDuration = 0;
+            } else if (utils.streamType(duration) === 'LIVE') {
+                let durationChanged = mediaDuration != duration;
+                if (durationChanged) {
+                    mediaDuration = duration;
+                    elapsedElem.innerText = 'Live';
+                    durationElement.innerText = '';
+                    sliderElem.style.width = '100%';
```

Comment:
I'm not particularly fond of handling it that way, what does the default receiver do?